packages/contracts-ops/contracts/test/RebootHome.t.sol

```diff
@@ -28,11 +30,16 @@ contract HomeRebootTest is RebootTest, HomeTest {
             coreAttributePath(localDomainName, "home.implementation")
         );
         reloadConfig();
-        pushSingleUpgrade(homeUpgrade(localDomainName));
+        pushSingleUpgrade(homeUpgrade(localDomainName), localDomainName);
         prankExecuteRecoveryManager(
             address(getGovernanceRouter(localDomainName)),
             getDomainNumber(localDomainName)
         );
+        // assert beacon has been upgraded to harness
+        (, bytes memory result) = address(homeUpgrade(localDomainName).beacon)
```

Comment:
should we make a convenience method for this on nomad test?

`getBeaconImplementation`

Reply:
`readBeacon`